src/chi/components/data-table/data-table.scss

```diff
@@ -0,0 +1,415 @@
+//     .chi-data-grid__column {
+//       display: flex;
+//       padding: .5rem;
+//
+//       @include respond-to(md) {
+//         border-bottom: 0.0625rem solid set-color(grey, 20);
+//         display: table-cell;
+//         padding: 1rem;
+//         vertical-align: top;
+//       }
+//
+//       &::before {
+//         content: attr(data-label);
+//         display: block;
+//         font-weight: 600;
+//         min-width: 10rem;
+//
+//         @include respond-to(md) {
+//           display: none;
+//         }
+//       }
+//     }
+//   }
+//
+//   &__row {
+//     border-bottom: 0.0625rem solid set-color(grey, 20);
+//     display: block;
+//
+//     @include respond-to(md) {
+//       background-color: set-color(white);
+//       border: 0;
+//       display: table-row;
+//     }
+//   }
+//
+//   &.-no-border {
+//     .chi-data-grid__head,
+//     .chi-data-grid__body {
+//       .chi-data-grid__column {
+//         @include respond-to(md) {
+//           border: 0;
+//         }
+//       }
+//     }
+//   }
+//
+//   &.-striped {
+//     .chi-data-grid__body {
+//       .chi-data-grid__row {
+//         &:nth-child(odd) {
+//           background-color: set-color(grey, 10);
+//         }
+//       }
+//     }
+//   }
+//
+//   &.-hover {
+//     .chi-data-grid__body {
+//       .chi-data-grid__row {
+//         &:hover,
+//         &.-hover {
+//           @include respond-to(md) {
+//             background-color: set-color(grey, 10);
+//           }
+//         }
+//       }
+//     }
+//   }
+//
+//   &.-two-columns-mobile {
+//     .chi-data-grid__body {
+//       @include respond-to(md) {
+//         display: table-row-group;
+//         vertical-align: middle;
+//       }
+//
+//       .chi-data-grid__column {
+//         display: flex;
+//
+//         @include respond-to(md) {
+//           border-bottom: 0.0625rem solid set-color(grey, 20);
+//           display: table-cell;
+//           padding: 1rem;
+//         }
+//
+//         &::before {
+//           display: inline-block;
+//           flex: 1 0 auto;
+//
+//           @include respond-to(md) {
+//             display: none;
+//           }
+//         }
+//       }
+//     }
+//
+//     .chi-data-grid__row {
+//       display: flex;
+//
+//       @include respond-to(md) {
+//         background-color: set-color(white);
+//         display: table-row;
+//       }
+//     }
+//   }
+// }
+
+.chi-data-table {
+  border-collapse: collapse;
+  display: block;
+  font-size: $text;
+  line-height: $line-height-smaller;
+  overflow-x: auto;
+  overflow-y: hidden;
+  width: 100%;
+
+  @include respond-to(md) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  label {
+    margin-bottom: 0;
+  }
+
+  &__head {
+    display: none;
+
+    @include respond-to(md) {
+      background-color: set-color(white);
+      display: block;
+    }
+
+    .chi-data-table__cell {
+      @include respond-to(md) {
+        border-bottom: 0.0625rem solid set-color(grey, 30);
```

Comment:
```suggestion
        border-bottom: 0.125rem solid set-color(grey, 30);
```

Reply:
this will fix the border width for all tables except `-bordered`. Need to define that separately.